frontend/www/js/omegaup/login/signin.ts

```diff
@@ -83,13 +95,26 @@ OmegaUp.on('ready', () => {
               ui.error(T.loginPasswordTooShort);
               return;
             }
-
-            api.User.create({
-              username: username,
-              email: email,
-              password: password,
+            const request: {
+              username: string;
+              email?: string;
+              dateOfBirth: Date;
+              parentEmail?: string;
+              password: string;
+              recaptcha: string;
+            } = {
+              username,
+              dateOfBirth,
```

Comment:
Acá esto fdebería quedar así:
```suggestion
              birth_date: dateOfBirth,
```